app/controllers/lightspeedauth_controller.rb

```diff
@@ -6,7 +6,11 @@ def index
     # Get the access token
     token = lth.trade_access_token(code)
     # write the current response to cache
-    Rails.cache.write("lightspeed_auth", token)
+    if Rails.env.development? || Rails.env.test?
+      File.write("#{Rails.root}/lightspeed_auth.json", token)
+    else
+      Rails.cache.write("lightspeed_auth", token)
+    end
```

Comment:
I'm not sure that I like this. It should work with the cache regardless of environment.